### DISCUSSION TITLE
Disable OTP for 2FA

### DISCUSSION THREAD

**Discussion by schack** [community/community#58795] [New to GitHub]
Hi
Is the possible to completely disable the OTP option for 2FA, I want to only be able to use security keys (Yubikeys)

**Comment by queenofcorgis** [comment]
Hi @schack, 

You do need two forms of authentication for 2FA, but there are [multiple options ](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) like GitHub Mobile authentication - you can make Yubikeys your [preferred method](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/changing-your-preferred-two-factor-authentication-method) if that helps.

**Reply by MightyBOBcnc** [reply]
It does not help, and it is ridiculous that we're forced to set up TOTP or SMS 2FA in order to use a hardware key that is more secure than both of those.

**Reply by adriankeenan** [reply]
> You do need two forms of authentication for 2FA

I only had OTP 2FA before adding my security keys, so github was clearly happy to run with a single 2FA method.

This requirement just doesn't make sense. After adding my security keys, the lowest security 2FA method for my account remains the same (OTP) - so what's the point?

**Comment by schack** [comment]
I would really like to get completely rid of the less secure authentication methods, minimizing the risk of successful phishing
